docs/odk-workflows/RepositoryFileStructure.md

```diff
@@ -34,6 +34,7 @@ These are the components in MONDO-INGEST
 | gard.owl | https://github.com/monarch-initiative/gard/releases/latest/download/gard.owl |
 | icd10cm.owl | https://data.bioontology.org/ontologies/ICD10CM/submissions/23/download?apikey=8b5b7825-538d-40e0-9e9e-5ab9274a9aeb |
 | icd10who.owl | https://github.com/monarch-initiative/icd10who/releases/latest/download/icd10who.ttl |
+| icd11.owl | https://icd11files.blob.core.windows.net/tmp/whofic-2023-04-08.owl.gz |
```

Comment:
## Other things in scope for this PR?
- [ ] @matentzn advise

@matentzn I added some subtasks to the OP that I'm not sure are needed. Can you let me and @twhetzel know if you think that these are in scope for (a) this PR, or (b) for #434, or (c) are rather for future potential PRs?
- `src/ontology/config/icd11_exclusions.tsv`?
- An exclude query for terms we don't want (sparql)?
- Any other considerations from #2?

Reply:
I would be happy if you add anything needed to run the whole pipeline successfully; and we iterate on the rest.

Reply:
But I would:

1. add an empty exclusion list
2. add a sparql query for selecting all diseases in the foundation (open in protege, find disease class, copy the DOID sparql query and just swap in the IRI of the "disease" class in the foundration)

Reply:
For _anything needed to run the whole pipeline successfully_, nothing comes to mind at the moment; perhaps it will become clear when running `build-mondo-ingest`, but if you think of anything specific, let me know. Otherwise, I added checkboxes above for myself~

Reply:
The exclusion list is created. I also created the SPARQL query(s) and confirmed one of them runs. Gonna open a separate thread about that.